infrastructure/lib/constructs/compliant-resources.ts

```diff
@@ -31,3 +32,33 @@ export class SecureBucket extends cdk.Construct {
     this.bucket = bucket;
   }
 }
+
+export interface PITRTableProps {
+  /**
+   * The properties to configure the DynamoDB table
+   */
+  tableProps: dynamodb.TableProps;
```

Comment:
```suggestion
  tableProps?: dynamodb.TableProps;
```

Let's make this optional

Reply:
Okay.

Reply:
The `tableProps` is now optional. When I made this change the table created in the class with `dynamobdb.Table` TypeScript was giving an error for the props. I did type casting to remove the error, but not sure if this was okay.

Reply:
ohh i see what the problem is. i'm sorry. i gave bad advice. i forgot that `partitionKey` is a required attribute and so therefore `TableProps` is required. do you mind reverting the `?` and the cast? sorry about that